src/lib/db.js

```diff
@@ -10,6 +11,8 @@ module.exports = () => {
     return new ArchivingDB(environment.archiveDestination);
   }
 
-  return new PouchDB(environment.apiUrl, { ajax: { timeout: 60000 } });
+  return new PouchDB(environment.apiUrl, {
+    ajax: { timeout: 60000 },
```

Comment:
I don't think this is a valid option anymore, I believe that this option was dropped when PouchDb switched to node-fetch from `pouchdb-ajax`.